src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/README.md

```diff
@@ -0,0 +1,15 @@
+## WasmBasicTestApp
+
+This a test application used by various Wasm.Build.Tests. The idea is to share a common behavior (so that we don't have to maintain many test apps) and tweak it for the test case.
```

Comment:
```suggestion
This is a test application used by various Wasm.Build.Tests. The idea is to share a common behavior (so that we don't have to maintain many test apps) and tweak it for the test case.
```